corehq/apps/reports/static/reports/js/filters/case_list_explorer.js

```diff
@@ -52,9 +56,11 @@ hqDefine("reports/js/filters/case_list_explorer", ['jquery', 'underscore', 'knoc
         });
 
         self.name.subscribe(() => {
-            var value = self._value();
-            if (value) {
-                self.label(value.label || self.name());
+            const value = self._value();
+            if (value && value.label) {
```

Comment:
It's not clear to me why you needed to change this from `(value)` to `(value && value.label)`. Isn't `value` either a property instance, or `null`? Or is it possible for a property instance not to have `value.label` set?

Reply:
It is true that `value` will either be a property instance or `null`. Given that both the CLE and DD are being GA'ed, all users that have the former should also have the latter. It therefore shouldn't be possible for `value` not to have a label. The reason I still added the check however, is that I rather opt to go on the safer side and not make that assumption, just in case there are edge case scenarios we are unaware of.

In saying the above, if you think that this is something we can safely take out I would be happy to change it back to `if  (value)`.

Reply:
Oh, no, I'm happy with it as it is. I was just wondering if there was something I wasn't considering.